fix: checkout -> check out (#2116)
getsentry/sentry-docs: src/docs/clients/cocoa/migration.mdx

  end
  ```

### Carthage

Remove `KSCrash`, as it is now bundled in the Sentry SDK.

  ```ruby
  github "getsentry/sentry-cocoa" "3"
  ```

### Public API

> We changed a lot of the public API, please check out [Advanced Usage](/clients/cocoa/advanced/#advanced) for more examples about that.

Reviewer: ```suggestion
Please check out our public API [Advanced Usage](/clients/cocoa/advanced/#advanced) for more examples.
```

## Upgrade from 3.x.x to 4.x.x {#upgrade-from-3-x-x-to-4-x-x}

- We removed the external `KSCrash` dependency and migrated the code directly into our SDK.

>   There shouldn't be any conflicts including `KSCrash`, as it's just not necessary anymore.

Reviewer: ```suggestion
  There shouldn't be any conflicts including `KSCrash`, as it's no longer necessary.
```
